reliquery/storage.py

```diff
@@ -846,9 +843,7 @@ def put_text(self, path: StoragePath, text: str, encoding="utf-8") -> None:
             self._update_binary_file(file_id, content)
 
     def get_text(self, path: StoragePath, encoding: str = "utf-8") -> str:
-        folder_id = self._find_deepest_folder_id(
-            self.root_id, [self.prefix] + path[:-1]
-        )
+        folder_id = self._find_deepest_folder_id(self.root_id, path[:-1])
         file_id = self._find_id_in_folder(folder_id, path[-1])
```

Comment:
interesting. I'm trusting you as the maintainer of this Storage that this makes sense.